aries_cloudagent/core/conductor.py

```diff
@@ -523,16 +548,20 @@ async def queue_outbound(
         # internal queue usually results in the message to be sent over
         # ACA-py `-ot` transport.
         if self.outbound_queue:
-            return await self._queue_external(profile, outbound)
+            await self._queue_external(profile, outbound)
         else:
-            return self._queue_internal(profile, outbound)
+            await self._queue_internal(profile, outbound)
+        return
```

Comment:
This is the end of the method, isn't it? Do we need the return here?

Reply:
no, but I thought it was nice to have. Say sayonara explicit return! I will remove it.